.github/workflows/release.yml

```diff
@@ -23,16 +22,8 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install virtualenv
-        make test package doc deploy
-    - name: Create Release
-      uses: actions/create-release@latest
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: ${{ github.ref }}
-        draft: false
-        prerelease: false
```

Comment:
I'm not sure how necessary we think the Github release is. Since we don't have a tag name, it would probably be a little challenging to get the release name from the `phc/version.py` file. If we think this is important though, I'm certainly willing to take a crack at it.